certbot-ci/certbot_integration_tests/utils/misc.py

```diff
@@ -209,3 +209,35 @@ def get_certbot_version():
     # Typical response is: output = 'certbot 0.31.0.dev0'
     version_str = output.split(' ')[1].strip()
     return LooseVersion(version_str)
+
+
+def find_certbot_root_directory():
```

Comment:
Instead of doing this, I'd rather make `certbot-ci` self contained.

To do this, I propose making something like a "testdata" directory within the `certbot-ci` package. After doing this, we can copy over `sample-config` in this directory and add it to `MANIFEST.in` for the package. We can then use `pkg_resources.resource_filename` to help us find the package. See our other packages for examples of all of this. Once all certbot-ci PRs have landed, we'll delete the `sample-config` found in `tests/integration`. 

I'm open to changing any of the details there, but I personally think keeping `certbot-ci` a little more self contained has value.

Reply:
In totally agree. I did not remember this solution at the time I wrote the test. I will do it now.

Reply:
Implemented. The config folder is put it `certbot_integration_tests/assets`, and marked in the manifest of certbot-ci to be included in the distribution.